src/backend/access/transam/remotexact.c

```diff
@@ -12,7 +12,8 @@
 #include "access/remotexact.h"
 
 /* GUC variable */
-int			current_region;
+bool multi_region;
```

Comment:
Need `extern bool multi_region;` in `remotexact.h`

Reply:
Done, sorry missed while copying the changes from the neon directory.